contracts/CourtRoom.sol

```diff
@@ -98,6 +129,9 @@ contract SwearGame is Owned {
 
 	}
 
+
+
+
 	function deposit(uint256 _depositAmount) onlyOwner payable public returns(bool){
```

Comment:
why not just use the default function which is called when you only transfer funds to a contract without data

`function()`